### PR TITLE
DR-1859 Add pools for TDR alpha, staging, perf, and prod

### PR DESCRIPTION
**Pull request by tlangs**
This PR needs to wait until https://github.com/broadinstitute/terraform-ap-deployments/pull/331 is merged so that permissions are correct.

### CODE REVIEW

**Review by samanehsan — APPROVED** [review]
Lgtm! ☔

**Review by snf2ye — APPROVED** [review]

File: src/main/resources/config/alpha/resource-config/datarepo_v1.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Data Repo buffered workspace template
+---
+configName: "datarepo_v1"
+gcpProjectConfig:
+  projectIdSchema:
+    prefix: "datarepo-alpha"
+    scheme: "RANDOM_CHAR"
+  parentFolderId: "270278425081" #data.test-terra.bio/repos/jade-dev
+  billingAccount: "01A74A-A76CA5-3F7A41"
```
Comment (yonghaoy) [inline src/main/resources/config/alpha/resource-config/datarepo_v1.yml:9]: We need to change this billing account if we agree to use the same one that WSM and Rawls are using

Comment (tlangs) [reply src/main/resources/config/alpha/resource-config/datarepo_v1.yml:9]: yes, I will do that now

File: src/main/resources/config/prod/resource-config/datarepo_v1.yml [hunk 2]
```diff
@@ -6,7 +6,7 @@ gcpProjectConfig:
     prefix: "datarepo-prod"
```
Comment (yonghaoy) [inline src/main/resources/config/prod/resource-config/datarepo_v1.yml:6]: WSM and Rawls does not have prod in prefix.  project in prod just `terra-`

**Review by yonghaoy — APPROVED** [review]

File: src/main/resources/config/prod/resource-config/datarepo_v1.yml (hunk 2 above)
Comment (yonghaoy) [reply src/main/resources/config/prod/resource-config/datarepo_v1.yml:6]: But that is up to you

Comment (tlangs) [reply src/main/resources/config/prod/resource-config/datarepo_v1.yml:6]: ehh I'll go with the convention. Lemme push.

**Review by yonghaoy — APPROVED** [review]

**Pull request merged**